cmd/crictl/container.go

```diff
@@ -111,6 +114,17 @@ var createPullFlags = []cli.Flag{
 		Value: "",
 		Usage: "Use `USERNAME` for accessing the registry. The password will be requested on the command line",
 	},
+	&cli.DurationFlag{
```

Comment:
I wonder if we need two of those or just one parameter `timeout` will be enough? Are those two having different semantics?